### DISCUSSION TITLE
Metadata Manager not available

### DISCUSSION THREAD

**Discussion by UMA-111** [combostrap/combo#74] [General]
Dear Gerard,
I try to start a new project for a documentation system based on combostrap and I am stuck already, because I can't launch the Metadata Manager.
I want to change the theme of a page, but this is then of course not possible.
When I click on the link for "Metadata Manager" I get following error message:
`Action unknown: metamanagermenuitem`
Something similar happens, btw with the Cache Manager:
`Action unknown: cachemenuitem`
Did I miss anything with the installation?

best regards,
Martin

**Comment by gerardnico** [comment]
Sorry for the late answer but for a reason that I need to investigate. The notification email went to the update tabs.

Combostrap is a progressive web app, meaning you should not change of page when performing an action.
We intercept the click and show a window. 

If this is not the case, this is because the page has not loaded completly.

If you give me an address where I can check, I can check myself. Otherwise, you need to check the network request for any failure (F12 in your browser).

**Comment by UMA-111** [comment]
Hi Gerard,
thanks for your answer and the hint to the right point: The js files are not loaded and generate an http 500 error. I will need to check on my server settings here.
thanks a lot!
cheers
Martin

**Reply by gerardnico** [reply]
Is it solved? 

I could may be add a meaningful text behind the action.

All the best

**Reply by UMA-111** [reply]
I did not dive deeper here, but switched completely to a different theme, where there were no issues with loading js files. Looks like some odd configuration on the hosting environment.

**Reply by gerardnico** [reply]
Okido. Thanks.

Installation in php is always a bing pain.

Have a nice week.
